test/micromamba/test_shell.py

```diff
@@ -29,7 +40,7 @@ def teardown_class(cls):
 
     @classmethod
     def setup(cls):
-        os.makedirs(TestShell.root_prefix, exist_ok=True)
+        os.makedirs(TestShell.root_prefix, exist_ok=False)
 
     @classmethod
     def teardown_class(cls):
```

Comment:
Duplicate  `teardown_class` definition

Reply:
I don't get the point here

Reply:
You have 2 `def teardown_class` definitions :) look at the full code.

Reply:
See for yourselves:

https://github.com/mamba-org/mamba/blob/c437573c61e3dda91bb2a429f457a85e0e5af799/test/micromamba/test_shell.py#L37-L47